draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -360,20 +343,37 @@ authorized(AS x, AS y) =  / Else, "Provider+" if the VAP-SPAS entry
         A valid path may have a one horizontal hop that represents peering relation.
       </t>
       <t>
-        <strong>Determination of Invalid AS_PATH:</strong>
```

Comment:
Renamed this because all cases were explained with the help of the newly introduced indices.